src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -4667,6 +4688,7 @@ private bool PerformCopyFileToRemoteSession(FileInfo file, string destinationPat
 
             bool targetSupportsAlternateStreams = RemoteTargetSupportsAlternateStreams(ps, remoteFilePath);
 
+#if !UNIX
```

Comment:
The PSCopyToSessionHelper script and PSCopyFromRemoteSession script for remote file copy contains support for alternate streams.  Ideally we would have versions of this script for Linux platforms that remove that support.  But I am not sure it should be part of this change.